chart/templates/mayastor/agents/core/agent-core-deployment.yaml

```diff
@@ -39,7 +39,7 @@ spec:
             requests:
               cpu: {{ .Values.agents.core.resources.requests.cpu | quote }}
               memory: {{ .Values.agents.core.resources.requests.memory | quote }}
-          image: "{{ .Values.image.registry }}/{{ .Values.image.repo }}/{{ .Chart.Name }}-agent-core:{{ .Values.image.tag }}"
+          image: "{{ .Values.image.registry }}/{{ .Values.image.repo }}/{{ .Chart.Name }}-agent-core:{{ default .Values.image.tag .Values.image.controlTag }}"
```

Comment:
suggesting thinking about some structure here, eg: .Values.image.tags.control-plane, .Values.image.tags.data-plane and .Values.image.tags.extensions ?

Reply:
Also please add them to values.yaml as "" with doc comments.

Reply:
A user may want to use it, in fact I already needed this a few weeks ago, I wanted discord user to test a change, to the control-plane, but there was no way to provide him with a nice way to do it via helm. (had to patch k8s deployments instead).
A production use case, some users (v1.0.x) upgrade control-plane more often than data plane, this would also provide them with a way to do so.

Reply:
Okay, sounds good then!

Reply:
@tiagolobocastro  : Do you mean we could have a section "tags" under image in values.yaml?
These tags will be used under main branch and hence I had these conveniently named

Reply:
Yes, having multiple tags at the root of images, just to keep it tidy - just a suggestion.
See what @niladrih thinks?

Reply:
Also this opens up an opportunity for a typical user to use Mayastor in un-tested ways.

Reply:
> Also this opens up an opportunity for a typical user to use Mayastor in un-tested ways.

That's their prerogative right? That happens now when users upgrade control-plane and not dataplane.

Reply:
Made changes as requested